app/assets/javascripts/records.js

```diff
@@ -94,6 +94,28 @@
         return clean;
       }
 
+      var filtersExist = function() {
+        if ($('.filter-btn').next().length > 0) { return true; }
+        else { return false; }
+      }
+
+      var makeApplyButton = function() {
+        if (filtersExist() === true) {
```

Comment:
This should be `if(filtersExist())` there is never any need to compare a boolean to true in an if statement